src/main/java/school/lemon/changerequest/java/vehicles/Vehicle.java

```diff
@@ -3,14 +3,59 @@
 /**
  * Created by lera on 06.12.16.
  */
-abstract class Vehicle {
-    protected int manufacturedYear;
-    protected String make;
-    protected String model;
+public class Vehicle implements IVehicle {
+    private int manufacturedYear;
+    private String make;
+    private String model;
 
     public Vehicle(String make, int manufacturedYear, String model) {
         this.make = make;
         this.manufacturedYear = manufacturedYear;
         this.model = model;
     }
+
+    @Override
+    public int getManufacturedYear() {
+        return manufacturedYear;
+    }
+
+    @Override
+    public void setManufacturedYear(int year) {
+        this.manufacturedYear = year;
+    }
+
+    @Override
+    public String getMake() {
+        return make;
+    }
+
+    @Override
+    public void setMake(String make) {
+        this.make = make;
+    }
+
+    @Override
+    public String getModel() {
+        return model;
+    }
+
+    @Override
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    @Override
+    public String accelerate() {
+        return null;
```

Comment:
There is no need to implement `Steerable` methods in this class.
Just make `Vehicle` abstract.